From 8647bda5e66198cb88fb34cf658a03e4368a19ce Mon Sep 17 00:00:00 2001
From: Marie-Cecile Afantenos <marie.cecile.afantenos@intel.com>
Date: Tue, 28 May 2013 09:45:29 +0200
Subject: [PDK MR2 ABSP] use new fstab format for MR2

BZ: 118706

This patch allow build script of MR2 to choose the second version of fstab for recovery that is only supported to MR2 and not MR1.
Signed-off-by: Marie-Cecile Afantenos <marie.cecile.afantenos@intel.com>

Change-Id: I0b1d8d0cca82dec7d2561af51729ba76679850df
---
 tasks/Recovery_Ota.mk |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/tasks/Recovery_Ota.mk b/tasks/Recovery_Ota.mk
index 913cde0..3f25fc2 100644
--- a/tasks/Recovery_Ota.mk
+++ b/tasks/Recovery_Ota.mk
@@ -18,6 +18,7 @@ recovery_resource_deps := $(shell find $(recovery_resources_common) \
   $(recovery_resources_private) -type f)
 recovery_modules := \
 	toolbox \
+	libselinux \
 	mksh \
 	strace \
 	linker \
@@ -281,6 +282,7 @@ endif
 	$(hide) $(ACP) $(APKCERTS_FILE) $(zip_root)/META/apkcerts.txt
 	$(hide)	echo "$(PRODUCT_OTA_PUBLIC_KEYS)" > $(zip_root)/META/otakeys.txt
 	$(hide) echo "recovery_api_version=$(PRIVATE_RECOVERY_API_VERSION)" > $(zip_root)/META/misc_info.txt
+	$(hide) echo "fstab_version=2" >> $(zip_root)/META/misc_info.txt
 ifdef BOARD_FLASH_BLOCK_SIZE
 	$(hide) echo "blocksize=$(BOARD_FLASH_BLOCK_SIZE)" >> $(zip_root)/META/misc_info.txt
 endif
-- 
1.7.4.1

